### DISCUSSION TITLE
Native Histogram Status

### DISCUSSION THREAD

**Discussion by SunSparc** [prometheus/prometheus#14111] [General]
Is there an estimate on when the `native histogram` feature will be moved out of the experimental phase?

I see in the docs and milestones that native histograms are still marked as experimental. 

- [Histogram Documentation](https://prometheus.io/docs/concepts/metric_types/#histogram)
- [Native Histograms milestone](https://github.com/prometheus/prometheus/milestone/10)

**Comment by beorn7** [comment]
Yes, native histograms are still not a stable feature (although it is less and less likely that anything substantially will change at this point).

The intention is to mark native histograms as stable in the upcoming v3 release of the Prometheus server, which is planned for September. But take that with a grain of salt. Things are done when they are done, not on a specific date.

**Reply by SunSparc** [reply]
Thank you for the update. I appreciate that adding a feature like this takes time to vet properly.
